src/software/export/main_exportAnimatedCamera.cpp

```diff
@@ -157,18 +261,42 @@ int aliceVision_main(int argc, char** argv)
       const std::string dstImage = (undistortedImagesFolderPath / (std::to_string(view.getIntrinsicId()) + "_" + imagePathStem + "." + image::EImageFileType_enumToString(outputFileType))).string();
       const camera::IntrinsicBase * cam = iterIntrinsic->second.get();
 
-      image::readImage(view.getImagePath(), image, image::EImageColorSpace::LINEAR);
+      image::readImage(view.getImagePath(), image, image::EImageColorSpace::NO_CONVERSION);
+      oiio::ParamValueList metadata = image::readImageMetadata(view.getImagePath());
 
       if(cam->isValid() && cam->hasDistortion())
       {
         // undistort the image and save it
         camera::UndistortImage(image, cam, image_ud, image::FBLACK, true); // correct principal point
```

Comment:
We need to compute the undistort image on the full RoD.

Reply:
Maybe we should add a dedicated option and always export in exr if `exportFullRoD` is enabled.

Reply:
It would be great to also expose the option to correct the principal point (which is always enabled for now).